composer.json

```diff
@@ -52,7 +52,7 @@
     },
     "require-dev": {
         "simplethings/entity-audit-bundle": "^0.9 || ^1.0",
-        "sonata-project/block-bundle": "^3.17",
+        "sonata-project/block-bundle": "^4.2",
```

Comment:
```suggestion
        "sonata-project/block-bundle": "^4.2",
```
```suggestion
        "sonata-project/block-bundle": "^3.20 || ^4.2",
```

Reply:
In v4 return type hints should be added. It mean parent with or without can be used. It will be much easier use master in 3.x branches.

Reply:
https://github.com/sonata-project/dev-kit/issues/780